build.gradle

```diff
@@ -116,6 +116,7 @@ configure(subprojects.findAll { it.name != 'util' }) {
         options.links 'http://docs.oracle.com/javase/7/docs/api/'
         options.tagletPath single(project(':util').sourceSets.main.output.classesDirs)
         options.taglets 'ManualTaglet'
+        options.taglets 'DochubTaglet'
```

Comment:
I find it odd that these are additive instead of overwriting each other.

Reply:
Agreed that is odd.